specs/results/results.spec

```diff
@@ -0,0 +1,23 @@
+PIPELINES-26
+# Results pvc tests
+
+## Test Tekton Results: PIPELINES-26-TC01
+Tags: results, sanity
+Component: Results
+Level: Integration
+Type: Functional
+Importance: Critical
+
+Steps:
+  * Ensure that tekton results is ready
+  * Create project "results-testing" 
+  * Apply in namespace "results-testing"
+    | S.NO | resource_dir                             |
+    |------|------------------------------------------|
+    | 1    | testdata/results/task-output-image.yaml  |
+  * Get "tr" logs and annotations
```

Comment:
Use taskrun/pipelinerun in this step, it will be more readable and easier to understand for new people